pkgs/top-level/all-packages.nix

```diff
@@ -1511,7 +1511,7 @@ in
 
   caddy = callPackage ../servers/caddy { buildGoModule = buildGo114Module; };  # https://github.com/lucas-clemente/quic-go/issues/2614
   caddy1 = callPackage ../servers/caddy/v1.nix { buildGoModule = buildGo114Module; };
-  traefik = callPackage ../servers/traefik { };
+  traefik = callPackage ../servers/traefik { buildGoModule = buildGo114Module; };
```

Comment:
Could you add a comment above buildGoModule explaining why it was pinned? This should help next maintainer to quickly know why without having to bisect.

Reply:
Good point, added!